src/renderer/components/Spinner.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div v-if="messageReceiving" 
+    :style="{
```

Comment:
Lets make this a computed property.

Reply:
Chris - Can I ask why this would be computed? This is a data field that I am just flipping on and off. Not sure what the computed prop would return other than this data field?

Reply:
It's inline with Vue conventions and will clean up the code. If its static, just make it a data prop.

Reply:
Fixed. Added two computed props - parentWidth + parentHeight